ddocs/medic-db/medic/nouveau/docs_by_replication_key/index.js

```diff
@@ -8,9 +8,17 @@ function (doc) {
       doc._id === 'privacy-policies' ||
       doc.type === 'form' ||
       doc.type === 'translations') {
-    return emit('_all', {});
+    index('string', 'key', '_all', { store: true });
+    return;
   }
-  var getSubject = function() {
+
+  var indexMaybe = (fieldName, value) => {
+    if (value) {
+      index('string', fieldName, value.toString(), { store: true });
```

Comment:
Since it looks like we only every query this index with `key:...`, I wonder if we should be using the `stored` field type for all of the non-key fields (`type`, `submitter`, etc)?  I assume that provides some performance/resource benefits over just using `string` for everything....

Reply:
>  I wonder if we should be using the [stored field type](https://docs.couchdb.org/en/stable/ddocs/nouveau.html#field-types) for all of the non-key fields (type, submitter, etc)?

I believe this enables the fields to be returned with the query result. It serves as the value from the couchdb function emit.

Reply:
No, I am not talking about the `{ store: true }` option.  (Which I agree functions as you said.)  I am talking about using `'stored'` as the first parameter instead of `'string'`. 

```js
index('stored', fieldName, value.toString());
```

I tested this and it seems to function as I expected.  When you index with the `stored` type, then the `{ store: true }` option _is implied_ and the field is returned in the `fields` object for the hit as desired.  However, you cannot _query_ against the values indexed with `stored`.  So, if I do `index('stored', 'needs_signoff', 'true');`, then I still get the `fields.needs_signoff` back on the hit, but I cannot query with `q: 'needs_signoff:true'` (which is fine since we never actually query with anything but `key`).